src/Features/Core/Portable/Navigation/NavigableItemFactory.SymbolLocationNavigableItem.cs

```diff
@@ -17,6 +19,7 @@ private class SymbolLocationNavigableItem : INavigableItem
             private readonly Solution _solution;
             private readonly ISymbol _symbol;
             private readonly Location _location;
+            private StrongBox<INavigableItem.NavigableDocument> _lazyDocument;
```

Comment:
Maybe doc this?  That null means, uncomputed, but we could compute and still point at null?

Reply:
⏱️ Enabling nullable reference types on this file to improve clarity

Reply:
➡️ Ended up documenting this, but not changing the nullability. There is a discrepancy in nullability which existed prior to this work, and I opted to defer fixing that.